lib/SIL/IR/TypeLowering.cpp

```diff
@@ -1453,6 +1453,7 @@ namespace {
 
       if (D->isCxxNotTriviallyCopyable()) {
         properties.setAddressOnly();
+        properties.setNonTrivial();
```

Comment:
Is it possible to factor out this fix into a separate PR and write a test for it?

Reply:
#32402 done :)

Reply:
Thanks! Now that PR is merged, would you mind rebasing to minimize the diff?